boost-maven/boost-maven-plugin/src/it/test-context/src/test/java/it/EndpointIT.java

```diff
@@ -0,0 +1,317 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package it;
+
+import static org.junit.Assert.*;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.apache.commons.httpclient.HttpClient;
+import org.apache.commons.httpclient.HttpStatus;
+import org.apache.commons.httpclient.methods.GetMethod;
+
+public class EndpointIT {
+    private static String URL;
+    private static Process process;
+
+    private static String HelloResource;
+    private static String osname;
+    private static boolean isWindows;
+    private static String originalStatement = "        return \"Hello World From Your Friends at Liberty Boost EE! start over - added the ctxRoot support and now added looseApp support - is it back?!?!?\";";
+    private static String updatedStatement = "        return \"Hello World From Your Friends at Liberty Boost EE! Updated after compile\";";
+
+    @BeforeClass
+    public static void init() {
+        URL = "http://localhost:9080/api/hello";
+
+        if (System.getProperty("os.name").contains("Windows")) {
+            isWindows = true;
+        } else {
+            isWindows = false;
+        }
+        setupHelloResources();
+
+    }
+
+    @Test
+    public void testServlet() throws Exception {
+        HttpClient client = new HttpClient();
+
+        GetMethod method = new GetMethod(URL);
+
+        try {
+            int statusCode = client.executeMethod(method);
+
+            assertEquals("HTTP GET failed", HttpStatus.SC_OK, statusCode);
+
+            String response = method.getResponseBodyAsString(10000);
+
+            assertTrue("Unexpected response body",
```

Comment:
Not that exact message. I just wanted to make sure the first part of the message was there in case something gets hosed up on the edit. No reason this test should fail as long as the first part of the message matches.